lib/rails_admin_import/formats.rb

```diff
@@ -0,0 +1,27 @@
+module RailsAdminImport
+  module Formats
+    class << self
+      def register(format, klass)
+        @registry[format.to_s] = klass
+      end
+
+      def for(format, *args)
+        @registry.fetch(format.to_s, DummyImporter).new(*args)
+      end
+
+      def all
+        @registry.keys
+      end
+
+      def reset
+        @registry = {}
+      end
+    end
+
+    reset
+  end
+end
+
+require 'rails_admin_import/formats/dummy_importer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.